org.eclipse.m2e.pde.connector/META-INF/MANIFEST.MF

```diff
@@ -13,5 +13,7 @@ Require-Bundle: org.eclipse.m2e.core;bundle-version="[2.0.0,3.0.0)",
  org.eclipse.core.resources;bundle-version="3.16.0",
  org.eclipse.m2e.maven.runtime;bundle-version="[3.8.6,4.0.0)",
  org.eclipse.pde.core,
- org.eclipse.jdt.core
+ org.eclipse.jdt.core,
+ org.eclipse.pde.ds.annotations;bundle-version="1.2.500",
```

Comment:
don't know which version range to take here

Reply:
Since you are using mainly internal classes any version range that does allow a future version has the potential to break.
IIRC in that case it is recommended to just specify a lower bound as you already did.

Reply:
In fact I only use constant which are interned at build time, so this is not a real run time requirement, but I haven't found a way to tell PDE to only load a package in the build classpath....